index.html

```diff
@@ -0,0 +1,139 @@
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>Kingsley's Portfolio</title>
+  </head>
+
+  <body class="background-color">
+    <header>
+      <nav class="nav-bar">
+        <ul>
+          <li>
+            <a href=""><div class="logo-font">Kingsfolio</div></a>
+          </li>
+          <li>
+            <a href=""><i class="fa fa-lg fa-bars menu-icon"></i></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="banner">
+      <div class="banner-background">
+        <img
+          class="illus-svg vector1"
+          src="assets/illustrations/vector-1.svg"
+          alt="###"
+        />
+        <img
+          class="illus-svg vector2"
+          src="assets/illustrations/vector-2.svg"
+          alt="###"
+        />
+        <img
+          class="illus-svg ellipse1"
+          src="assets/illustrations/Ellipse-15.svg"
+          alt="###"
+        />
+        <img
+          class="illus-svg vector3"
+          src="assets/illustrations/vector-3.svg"
+          alt="###"
+        />
+        <img
+          class="illus-svg vector5"
+          src="assets/illustrations/vector-5.svg"
+          alt="###"
+        />
+        <img
+          class="illus-svg vector4"
+          src="assets/illustrations/vector-4.svg"
+          alt="###"
+        />
+        <img
+          class="illus-svg ellipse2"
+          src="assets/illustrations/Ellipse-15.svg"
+          alt="###"
+        />
+      </div>
+      <div class="banner-stack">
+        <h1 class="banner-header banner-content">
+          I'm Kingsley, Glad to see you!
+        </h1>
+
+        <p class="body-text-2 banner-content">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+
+        <p class="head-text-1 banner-content">LET'S CONNECT</p>
+
+        <div class="banner-content banner-socials">
+          <ul class="banner-socials">
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/twitter.svg"
+                  alt="Twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/linkedin-in.svg"
+                  alt="Linkedin"
```

Comment:
- [ ] As requested by the previous reviewer, kindly change the colors of the social media icons to match those of the design.
Your Design
![image](https://user-images.githubusercontent.com/81932478/170447900-f5d10e64-715b-41ba-b193-24e7e332cd2c.png)

Figma Design
![image](https://user-images.githubusercontent.com/81932478/170449934-fc42990c-7432-46ba-9f4c-6de2ec5db029.png)

Reply:
Hi @Munsa1 ,

I already changed the icons in the repo, you're not supposed to see these black icons. I just confirms that the new icons are there. Please confirm that the new icons are there.

Regards